docs/Resources.md

```diff
@@ -28,6 +28,33 @@ The `rootfs_data` (formerly `userfs`) partition holds whatever file change (conf
 
 When a proper [Reset to Factory Defaults (RTFD)](../Recovery/#reset-to-factory-defaults-rtfd) is done, the overlay partition is not formatted, the only relevant `/overlay/bank_*` folder is deleted instead. You can learn more on such aspects by reading the [Recovery](../Recovery/) page.
 
+### Devices with U-boot bootloader
+
+Here is how the Homeware flash layout typically looks like on newer ARM64 boards:

```

Comment:
Better to avoid mentioning ARM64 here, there are some devices which are ARM64 but still having traditional layout.

We should study these new firmware base image sdk and find a distinctive name for them which somehow recaps the U-Boot/UBIFS/PKGTB touple.

Reply:
Sorry - didn't know there were other ARM64 devices. I have fixed to just reference U-boot bootloader.